backends/kiibohd.py

```diff
@@ -244,11 +251,20 @@ def process( self, capabilities, macros, variables, gitRev, gitChanges ):
 					# TODO Add support for Analog keys
 					# TODO Add support for LED states
 					self.fill_dict['TriggerMacros'] += "0x00, 0x01, 0x{0:02X}, ".format( triggerItem )
+					triggers_count += 1
```

Comment:
You should be able to just do:
`triggers_count =len( macros.triggersIndexSorted )`
You're counting each element of the sequence of combos, which may be a lot higher than you want.

However, if you have a sequence of over 255 combos, it may also be useful to increase StateWordSize to 16 or even 32 if 65k isn't enough.
At around line 236 do:
`trigger_sequence_count += len( macros.triggersIndexSorted[ trigger ][0] )`